packages/oui-action-menu/src/action-menu-item.directive.js

```diff
@@ -0,0 +1,54 @@
+import controller from './action-menu-item.controller'
+
+const baseClass = 'oui-action-menu__item'
+const dividerClass = 'oui-action-menu__item_divider'
+const buttonClasses = 'oui-button oui-button_action-menu'
+
+const linkTemplate = (dividerClass, external) => `
+  <li class="${baseClass} ${dividerClass}">
+    <a
+      class="${buttonClasses}"
+      data-ng-href="{{::$ctrl.href}}">
+      <span data-ng-bind="::$ctrl.text"></span>
+      ${external ? '<i class="oui-icon oui-icon-external_link" aria-hidden="true"></i>' : ''}
+    </a>
+  </li>`
+
+const buttonTemplate = dividerClass => `
+  <li class="${baseClass} ${dividerClass}">
+    <button
+      class="${buttonClasses}"
+      type="button"
+      data-ng-click="$ctrl.onClick()"
+      data-ng-bind="::$ctrl.text"></button>
+  </li>`
+
+export default $compile => {
```

Comment:
I respect ESLint rules.

Reply:
@jleveugle all ovh-ui-kit related project has it own eslint rules because we didn't had the ovh eslint rules and we do not have had time to integrate and refactor all those projects. We know it should and will be done in futur.